SU-SWS/react_paragraphs
# D8CORE-2686 Paragraph and row behaviors UI

# READY FOR REVIEW

# Summary
- Added paragraph behaviors UI to the widget.
- Adjusted style of the viewfield component

# Need Review By (Date)
- 11/4

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. Use the example file `react_paragraphs_behaviors.react_behaviors.yml`, copy it to a module/theme and rename appropriately & uncomment the definitions.
1. clear drupal and browser caches
1. Create a basic page and add a paragraph type and row that you've defined in the plugin definition above.
1. save with various "Styles" values.
1. create a test hook `hook_preprocess_paragraph` or `hook_preprocess_paragraphs_row`
1. inspect the entity in the `$variables` argument and verify the values you chose in the "Styles" area from the UI are represented in the entity.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
